test/jasmine/tests/geo_test.js

```diff
@@ -1150,9 +1150,22 @@ describe('Test geo interactions', function() {
                     done();
                 });
             });
-
         });
     });
+
+    it('should not throw during hover when out-of-range pts are present in *albers usa* map', function(done) {
```

Comment:
which :white_check_mark: https://github.com/plotly/plotly.js/pull/2030#issuecomment-331560323

Note that I kept the `scattergeo-out-of-usa` because the image test does :lock: down the bug fix.

See the rogue point(s) in the upper-left corner:

![image](https://user-images.githubusercontent.com/6675409/30827823-71fe87aa-a209-11e7-8255-a066c5e5e4cd.png)


this thing :arrow_double_down: 

![image](https://user-images.githubusercontent.com/6675409/30827832-76fa2c6e-a209-11e7-86ec-5de1f9baa4d5.png)

But yeah, adding a jasmine test documents the behavior better, so here it is.